README.md

```diff
@@ -1,5 +1,22 @@
 This [dbt](https://github.com/fishtown-analytics/dbt) package contains macros that can be (re)used across dbt projects.
 
+## Getting started
+
+To use dbt-utils:
+
+1. Reference the package in your dbt project by creating a `packages.yml` file, which should contain the following:
+
+  ```
+  packages:
+    - git: "https://github.com/fishtown-analytics/dbt-utils.git"
+      revision: master
```

Comment:
we typically advise pinning your dep to a specific release. Can you change `master` to something like `0.1.22` and add a comment to consult https://github.com/fishtown-analytics/dbt-utils/releases for information about the latest release?

Reply:
Ah, _that's_ what you meant when you said "we need to bump it every time we make a new release". It makes total sense though. I'll update the PR.

For what it's worth, this recommendation is nowhere to be found in the [Package Management](https://docs.getdbt.com/docs/package-management) section. The example given there is pointing to master.